util/Migrator/DbScripts/2025-08-29_00_CipherArchiveInit.sql

```diff
@@ -0,0 +1,597 @@
+-- Add the ArchivedDate column to the Cipher table
+IF COL_LENGTH('[dbo].[Cipher]', 'ArchivedDate') IS NULL
```

Comment:
❓ @bitwarden/dept-dbops Are there any special considerations we should take when modifying the `Cipher` table/indexes?